compiler/rustc_mir_build/src/builder/expr/as_rvalue.rs

```diff
@@ -572,6 +572,14 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
                 );
                 block.and(Rvalue::Use(operand))
             }
+
+            ExprKind::ByUse { expr, span: _ } => {
+                let operand = unpack!(
+                    block =
+                        this.as_operand(block, scope, expr, LocalInfo::Boring, NeedsTemporary::No,)
```

Comment:
```suggestion
                        this.as_operand(block, scope, expr, LocalInfo::Boring, NeedsTemporary::No)
```

rustfmt doesn't format macro invocations.